plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "./models/ExitModel.sol";
+import "./interfaces/ExitProcessor.sol";
+import "./registries/ExitGameRegistry.sol";
+import "./priorityQueue/PriorityQueue.sol";
+
+contract ExitGameController is ExitGameRegistry {
+    uint64 public exitQueueNonce = 1;
+    mapping(uint256 => ExitModel.Exit) public exits;
+    mapping (address => PriorityQueue) public exitsQueues;
```

Comment:
Don't we want to keep those queues private?

Reply:
haha, was being lazy and this would be top contract anyway (supposedly only `PlasmaFramework` would inherit this and would be nothing more than a contract that simply inherits multiple contract to get all functions)

updated with private vars.